### ISSUE TITLE
Cannot compile contract with scientific notation literals: "Invalid numeric literal ..."

### ISSUE THREAD

**Issue by andreibancioiu** [second-state/SOLL#8]
Not sure if already reported in the past, but using the scientific notation literal in a contract results in a compilation error.

**Example contract:**

https://solidity.readthedocs.io/en/v0.5.10/introduction-to-smart-contracts.html#subcurrency-example

```
function mint(address receiver, uint amount) public {
    require(msg.sender == minter);
    require(amount < 1e60);
    balances[receiver] += amount;
}
```

**Error:**

```
Invalid numeric literal '1e60'.
```

**Comment by hydai** [comment]
Thanks for reporting.
We haven't implemented scientific notation literal type. This is on our future plan.

**Comment by hydai** [comment]
Fixed at fc38fd053ffe31f5890593be367eff5b5d6af478
